src/views/components/dataset-navigation.html

```diff
@@ -25,6 +25,11 @@
         text: "Dataset overview",
```

Comment:
This should be "Dataset details" — it was implemented incorrectly in the original commit